src/components/forms/DomainSettings/TTLRow.tsx

```diff
@@ -84,7 +88,21 @@ export default function TTLRow({
         editable={editable}
         editing={editing}
         setEditing={() => setEditing(true)}
-        onSave={() => setShowModal(true)}
+        onSave={() => {
+          try {
+            if (newTTL < MIN_TTL_SECONDS)
+              throw new Error(
+                `TTL must be greater than ${MIN_TTL_SECONDS.toLocaleString()}`,
```

Comment:
why not use our constant?

Reply:
Can you clarify? I am using our constant

Reply:
https://github.com/ar-io/arns-react/pull/661/files#r1940098745

Reply:
[6f6829b](https://github.com/ar-io/arns-react/pull/661/commits/6f6829b3ebec9cd66571a9f8c7d80546ce87a8bc)